Presentation.Web/App_Start/MappingConfig.cs

```diff
@@ -355,6 +360,7 @@ public MappingProfile()
             CreateMap<ItContract, ItContractDTO>()
                   .ForMember(dest => dest.AssociatedSystemUsages, opt => opt.MapFrom(src => src.AssociatedSystemUsages.Select(x => x.ItSystemUsage)))
                   .ForMember(dest => dest.AgreementElements, opt => opt.MapFrom(src => src.AssociatedAgreementElementTypes.Select(x => x.AgreementElementType)))
+                  .ForMember(dest => dest.CriticalityName, opt => opt.MapFrom(src => src.CriticalityType.Name))
```

Comment:
I think you need to look at how the other choice types are mapped because CriticalityName is not enough so no need to spell it out